libs/sdk-backend-spi/src/workspace/fromModel/ldm/metadata/measure/index.ts

```diff
@@ -29,3 +54,13 @@ export interface IMeasureMetadataObject extends IMetadataObject {
 export function isMeasureMetadataObject(obj: unknown): obj is IMeasureMetadataObject {
     return isMetadataObject(obj) && obj.type === "measure";
 }
+
+/**
+ * Tests whether the provided object is of type {@link IMeasureMetadataObjectDefinition}.
+ *
+ * @param obj - object to test
+ * @public
+ */
+export function isMeasureMetadataObjectDefinition(obj: unknown): obj is IMeasureMetadataObject {
+    return (obj as IMetadataObject).type === "measure" && (obj as IMetadataObject).ref === undefined;
```

Comment:
This could probably use the `isMeasureMetadataObject` from above for the first part of the check?

Reply:
@no23reason `isMeasureMetadataObject` calls `isMetadataObject` which checks ref, but the definition does not contain ref.